infra/examples/gcp-google-workspace/main.tf

```diff
@@ -138,7 +142,8 @@ module "google-workspace-connection" {
 module "google-workspace-connection-auth" {
   for_each = local.enabled_google_workspace_sources
 
-  source = "git::https://github.com/worklytics/psoxy//infra/modules/gcp-sa-auth-key-secret-manager?ref=v0.3.0-beta.5"
+  source = "../../modules/gcp-sa-auth-key-secret-manager"
+  # source = "git::https://github.com/worklytics/psoxy//infra/modules/gcp-sa-auth-key-secret-manager?ref=v0.3.0-beta.5"
```

Comment:
do you plan to leave all the examples pointing to local instead of a fixed version? why?

Reply:
no, should be fixed to remote